### PR TITLE
Remove and deprecate unused variables and correct queueDepth default in IBM MQ scaler

### PR DESCRIPTION
**Pull request by rickbrouwer**
- The current documentation states that the default `queueDepth` is `5`, while the code default is `20`. The documentation will be updated to reflect the correct default value of `20`, aligning it with the code.
- The `queueManager` parameter is currently checked to see if it's filled, but it's not used elsewhere in the code. This unused code is removed. The documentation is updated accordingly.
- The parameter `tlsDisabled` is set to **DEPRECATED**. All other scalers use the parameter `unsafeSsl`. To maintain consistency across the other scalers, the code related to `tlsDisabled` is set to deprecated, as `unsafeSsl` already handles this functionality.
- Finally, some minor adjustments to the documentation to keep the formatting the same.

### Checklist

- [X] Commits are signed with Developer Certificate of Origin (DCO)

Related: [6034](https://github.com/kedacore/keda/pull/6034)

### CODE REVIEW

**Comment by JorTurFer** [comment]
Are all deprecated?
image

I thought that only the tls one is deprecated

File: content/docs/2.16/scalers/ibm-mq.md [hunk 1]
```diff
@@ -16,27 +16,25 @@ triggers:
 - type: ibmmq
   metadata:
     host: <ibm-host> # REQUIRED - IBM MQ Queue Manager Admin REST Endpoint
-    queueManager: <queue-manager> # REQUIRED - Your queue manager
     queueName: <queue-name> # REQUIRED - Your queue name
-    tlsDisabled: <TLS enabled/disabled> # OPTIONAL - Set 'true' to disable TLS. Default: false
-    queueDepth: <queue-depth> # OPTIONAL - Queue depth target for HPA. Default: 5 messages
+    tlsDisabled: <TLS enabled/disabled> # DEPRECATED: This parameter is deprecated as of KEDA v2.16 in favor of unsafeSsl and will be removed in version v2.17
```
Comment (JorTurFer) [inline content/docs/2.16/scalers/ibm-mq.md:20]:
```suggestion
    tlsDisabled: <TLS enabled/disabled> # DEPRECATED: This parameter is deprecated as of KEDA v2.16 in favor of unsafeSsl and will be removed in version v2.18
```

File: content/docs/2.16/scalers/ibm-mq.md [hunk 2]
```diff
@@ -16,27 +16,25 @@ triggers:
...
 
 **Parameter list:**
 
 - `host` - IBM MQ Queue Manager Admin REST Endpoint. Example URI endpoint structure on IBM cloud `https://example.mq.appdomain.cloud/ibmmq/rest/v2/admin/action/qmgr/QM/mqsc`.
-- `queueManager` - Name of the queue manager from which messages will be consumed.
-- `queueName` - Name of the Queue within the Queue Manager defined from which messages will be consumed.
-- `tlsDisabled` - Can be set to 'true' to disable TLS. (Values: `true`, `false` , Default: `false`, Optional)
-- `queueDepth` - Queue depth Target for HPA. (Default: `5`, Optional)
+- `queueName` - Name of the Queue defined from which messages will be consumed.
+- `tlsDisabled` - Can be set to 'true' to disable TLS. (DEPRECATED: This parameter is deprecated as of KEDA v2.16 in favor of unsafeSsl and will be removed in version v2.17, Values: `true`, `false` , Default: `false`, Optional)
```
Comment (JorTurFer) [inline content/docs/2.16/scalers/ibm-mq.md:32]:
```suggestion
- `tlsDisabled` - Can be set to 'true' to disable TLS. (DEPRECATED: This parameter is deprecated as of KEDA v2.16 in favor of unsafeSsl and will be removed in version v2.18, Values: `true`, `false` , Default: `false`, Optional)
```

**Comment by rickbrouwer** [comment]
> Are all deprecated? image
> 
> I thought that only the tls one is deprecated

Yes, sorry. My bad. I misunderstood something

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**